lighthouse-core/report/html/report-styles.css

```diff
@@ -811,7 +812,7 @@
 
 .lh-warnings {
   --item-margin: calc(var(--body-line-height) / 6);
-  border: 1px solid var(--average-color);
+  border: 1px solid var(--color-orange);
```

Comment:
what's everyone's opinions on color variable names?

their logical usage like `average-color` or what they are like `black-400`?

Reply:
I just copied these from yuinchen's design, but I'll share what I think about them:

* starting with `color-` is great for autocomplete
* `color-xxx` is neat, IDK what the scale represents tho.
* using `orange` over `average` is good for quick-understanding, but bad for theming (do we care?)

Reply:
@Hoten FYI, the color scale is from material color system: https://material.io/design/color/the-color-system.html#tools-for-picking-colors. 500 is the primary, above is darker, below is lighter.

Reply:
I'm happy to defer to others on this, but to me `--average-color` tells me how to use it, while `--color-orange` is something I'll have to look up when I'm trying to figure out what to use for the average color.

There's limits to the expressiveness of variable names (who knows what `--report-secondary-border-color` really refers to :), but it does seem useful to learn more towards how it's used rather than what it contains.

Reply:
ya i'm with ya on this. done.